stylus/components/forms.styl

```
@@ -269,6 +269,16 @@ $input-text
     &:invalid
         border rem(1) solid pomegranate
 
+    &[aria-disabled]
```

```suggestion
    &[aria-disabled]
    &[disabled]
```
For uses outside of react components, if there's no `aria-disabled`, which is ok beacuse it's kinda redundant since most screen readers understand the `disabled` attribute just as well.
But I'll keep both, it doesn't cost much and covers all cases.

---

Thanks !